codingStandards/CodingStandard-Css.md

```diff
@@ -0,0 +1,242 @@
+# CSS Coding Standard
+
+## Contents
+ - [General](#general)
+ - [CSS Files](#css-files)
+ - [Selectors](#selectors)
+   - [Selector Formatting](#selector-formatting)
+   - [Use of Efficient Selectors](#use-of-efficient-selectors)
+ - [Classes](#classes)
+   - [Naming Standards](#naming-standards)
+ - [Attributes](#attributes)
+   - [Order of Attributes](#order-of-attributes)
+   - [Attribute Formatting](#attribute-formatting)
+   - [General Details](#general-details)
+ - [Resources](#resources)
+   - [Highly Recommended](#highly-recommended)
+   - [Other Readings](#other-readings)
+
+## General
+
+* Use the [Google CSS Style Guide](https://google-styleguide.googlecode.com/svn/trunk/htmlcssguide.xml)
+  for any topics not covered in this document.
+
+* If the project is using a framework (e.g. Bootstrap/jQuery), use classes from the framework as much as possible.
+
+* Divide long css files into sections. 
+
+ > By segmenting the file(s) in this logical way, it is easy to locate css rules and to ensure that a new rule
+ > being added is not already present.
+
+
+## CSS Files
+
+* CSS file includes must be done using `<link>` tags in the html/jsp files and NOT using `@import` in other css files.
+  
+  > **`@import`** can be slow and may result in the page being rendered without css for a while before the page 
+  > magically comes to life. As of now we are using css files as stylers of html pages and thus the dependency 
+  > must be clearly visible in the html page.
+
+  :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :arrow_heading_down:
```

Comment:
I'm trying to get rid of that colored Good/Bad thing. The whole additional HTML file exists just for that thing. Without it, we can do with the `.md` file only. Yes, this one doesn't look as nice and we cannot have good and bad, side-by-side, but if we can do with just the markdown file it makes the markdown file cleaner (i.e. no `<table>` tags), loading faster, and no need for an extra html file.? What do you think?